pptx_blueprint/__init__.py

```diff
@@ -5,6 +5,11 @@
 __Pathlike = Union[str, pathlib.Path]
 
 
+class TemplateOvewriteError(Exception):
+    """An error that occurs when the user tries to override the template."""
+    pass
+
+
 class Template:
```

Comment:
We should wait for the final implementation of this class, but given my current understanding of this class, it should be either a PresentationAdapter or a PresentationFacade, because what we are doing here is replacing the somewhat unhandy interface of pptx.Presentation with an interface more suitable for our purposes.

Reply:
Resolving this since it's not in the scope of this PR

Reply:
Agreed, I just wanted to use this PR as a platform so raise awareness / start a discussion :-)